components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -1594,6 +1594,13 @@ initialize (const ekat::Comm& atm_comm,
 void AtmosphereDriver::run (const int dt) {
   start_timer("EAMxx::run");
 
+  // DEBUG option: Check if user has set the run to fail at a specific timestep.
+  if (m_atm_params.isSublist("driver_debug_options")) {
```

Comment:
You don't need to check if a sublist exists, imho. If you grab it, but did not exist, it will get created (empty). But adding the debug sublist is not really an issue, and arguably the code is easier to parse.

And I agree with Naser, that the line below is prob too long. You could do this:
```c++
auto& debug = m_atm_params.sublist("driver_debug_options");
auto fail_step = debug.get<int>("force_fail_at_step",-1);
if (fail_step==m_current_ts.get_num_steps()) {
  std::abort();
}
```